platform/ios/src/MGLMapboxEvents.m

```diff
@@ -81,22 +83,22 @@ - (void)userDefaultsDidChange:(NSNotification *)notification {
 }
 
 - (void)updateNonDisablingConfigurationValues {
-    self.eventsManager.debugLoggingEnabled = [[NSUserDefaults standardUserDefaults] boolForKey:@"MGLMapboxMetricsDebugLoggingEnabled"];
+    self.eventsManager.debugLoggingEnabled = [[NSUserDefaults standardUserDefaults] boolForKey:MGLMapboxMetricsDebugLoggingEnabledKey];
     
     // It is possible for `MGLTelemetryAccessToken` to have been set yet `userDefaultsDidChange:`
```

Comment:
Nit: `MGLTelemetryAccessToken` - > `MGLTelemetryAccessTokenKey`

Reply:
Hrm, yeah... these comments should probably not refer to either the variable name or the key string.